### PR TITLE
TIM - Extracting URLs with complex numerical IPs

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-47736
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-48624

## Description
Updating the URL regex and formatter to support edge cases with numerical IPs.
New regex tests:
1. https://regex101.com/r/CWfo3A/1
2. https://regex101.com/r/1hgnpN/2

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_19_32.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### FormatURL
+
+- Updated the FormatURL script to support more complex cases of numerical IPs in URLs.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_19_32.md:6]:
```suggestion
Updated the **FormatURL** script to support more complex cases of numerical IPs in URLs.
```

File: Packs/Phishing/ReleaseNotes/3_6_38.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### FindDuplicateEmailIncidents
+
+- Updated the FindDuplicateEmailIncidents script to handle more cases of numerical IPs in URLs.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_38.md:6]:
```suggestion
Updated the **FindDuplicateEmailIncidents** script to handle more cases of numerical IPs in URLs.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @MLainer1 Doc review completed.

**Review by eyalpalo — APPROVED** [review]
